app/views/achievements/index.html.haml

```diff
@@ -1,4 +1,4 @@
 .row
-  .col-xs-12
+  .col-md-12
```

Comment:
I'm just lazily using Bootstrap's grid system to cope with responsiveness. I and occasionally others will pull it up on a phone (e.g. in the shop), so it's good for it to suit a narrow viewport.